packages/wallets/src/components/WalletHistory.tsx

```diff
@@ -192,6 +192,8 @@ type Props = {
   walletId: number;
 };
 
+const mapMemos: any = {};
```

Comment:
I would have used `useState({})` to avoid creating a global variable (bad)

Reply:
I looked into using the LRU code, but since this component is in the wallets package, it would've involved moving a bunch of code. I'm ok with this being a global for now.